GomokuClient/packages/gomoku-core/src/main.tsx

```diff
@@ -65,7 +65,9 @@ createRoot(document.getElementById("root")!).render(
           <ClerkProvider publishableKey={PUBLISHABLE_KEY} afterSignOutUrl="/">
             <AuthTokenProvider>
               <ToasterProvider>
-                <RouterProvider router={router} />
+                <SignalRProvider>
+                  <RouterProvider router={router} />
+                </SignalRProvider>
```

Comment:
Let’s move `<SignalRProvider/> `higher in the component hierarchy so that notification systems like `<ToasterProvider/>` are nested beneath it. This allows SignalRProvider to provide notifications to the rest of the app. Let’s place it right after `<AuthTokenProvider/> `please